pkgs/desktops/gnome-3/3.16/core/gnome-control-center/default.nix

```diff
@@ -17,7 +18,8 @@ stdenv.mkDerivation rec {
     sha256 = "07vvmnqjjcc0cblpr6cdmg3693hihpjrq3q30mm3q68pdyfzbjgf";
   };
 
-  propagatedUserEnvPkgs = [ gnome3.gnome_themes_standard gnome3.libgnomekbd ];
+  propagatedUserEnvPkgs =
+    [ gnome3.gnome_themes_standard gnome3.libgnomekbd sound-theme-freedesktop ];
```

Comment:
No, sorry, I don't know *why*, just *that*. I added it to `XDG_DATA_DIRS` first, noticed it still didn't work, then added it to `propagatedUserEnvPkgs` and it did.

Interestingly, it's the `XDG_DATA_DIRS` addition that turns out not to make a difference. It still works without it, once I reboot.

I'd like to be of more help but that will require me finally figuring out how the Gnome 3 environment is composed. I'd been able to avoid that so far. Oh well. I'll dive in this evening.